### DISCUSSION TITLE
Gamma correction of text on Linux

### DISCUSSION THREAD

**Discussion by madig** [ghostty-org/ghostty#4060] [Q&A] [answered]
Hi there,
I stumbled over https://github.com/ghostty-org/ghostty/issues/2122 and https://github.com/ghostty-org/ghostty/issues/2125 and was wondering if implementing gamma correction would also affect text rendering on Linux? I noticed that text looks a bit "darker" than on the default GNOME Terminal. Doing proper gamma correct and linear alpha blending would make ghostty look non-native on Linux ([because every toolkit does it wrong, except Qt with CFF fonts](https://freetype.org/freetype2/docs/hinting/text-rendering-general.html)), but I would welcome it :)

**Comment by pluiedev** [comment, marked as answer]
It would affect the text rendering, yes, since that's part of the renderer. Further questions should go to #2125
